pkg/controller/lifecycle/actuator_test.go

```diff
@@ -246,10 +246,9 @@ webhooks:
 `
 }
 
-func expectedSeedValidatingWebhook(caBundle []byte, namespace string, failurePolicy admissionregistrationv1.FailurePolicyType) string {
+func expectedSeedValidatingWebhook(caBundle []byte, namespace string) string {
 	var (
-		caBundleEncoded  = b64.StdEncoding.EncodeToString(caBundle)
-		strFailurePolicy = string(failurePolicy)
+		caBundleEncoded = b64.StdEncoding.EncodeToString(caBundle)
```

Comment:
```suggestion
		caBundleEncoded := b64.StdEncoding.EncodeToString(caBundle)
```

Reply:
This assignment operator is not allowed inside `var ()` block 😉 

```sh
$ make test
...
# github.com/gardener/gardener-extension-shoot-lakom-service/pkg/controller/lifecycle
pkg/controller/lifecycle/actuator_test.go:251:19: expected type, found ':='
```

Reply:
I meant removing the `var` keyword since it is a single variable in there and there is no need for grouping

Reply:
Done with e1c2bcb7